docs/README.md

```diff
@@ -462,7 +462,30 @@ Validate pull requests for mergeability based on content and structure of your P
   ```
   </p>
 </details>
+<br>
+
+**Size**: Ensure that PRs don't exceed a certain size (in terms of lines changed). Currently this just
+checks the combined total of additions and deletions across all files that were added or modified.
+It ignores files that were completely deleted. You can specify a list of files to
+ignore with `ignore`.
 
+<details><summary>🔖 See Recipe</summary>
+  <p>
+
+  ```yml
+  version: 2
+  mergeable:
+    - when: pull_request.*
+      validate:
+        - do: size
+          ignore: ['ignore_me.js']
+          max:
```

Comment:
Can i suggest having 
```
lines:
  max: ...
``` 
instead? i can imagine a case where size of an PR is determined by the number of files affected as well.
The feature to check PR size by files could be implemented later.

Reply:
Ah, good idea! I've updated it to follow that structure instead.